packages/docusaurus-plugin-content-blog/src/blogUtils.ts

```diff
@@ -242,8 +243,17 @@ async function processBlogSourceFile(
     } else if (parsedBlogFileName.date) {
       return parsedBlogFileName.date;
     }
-    // Fallback to file create time
-    return (await fs.stat(blogSourceAbsolute)).birthtime;
+
+    try {
+      const result = getFileCommitDate(blogSourceAbsolute, {
+        age: 'oldest',
+        includeAuthor: false,
+      });
+      return result.date;
+    } catch (e) {
+      logger.error(e);
```

Comment:
@slorber @felipecrs I was tracking why I started to see ERROR on my builds, this catch is causing it. In verdaccio I use crowdin and I do no thave all files commited in the git repository since are downloaded in build time. I was wondering if we can open a discussion about this, in my case does not break the build but polute and misslead a bit, I thought was an error until I read the soure code, what do you think?

As an example, I am not sure if you can see the logs here https://github.com/verdaccio/verdaccio/runs/5974881123?check_suite_focus=true

```
[ERROR] Error: Failed to retrieve the git history for file "C:\Users\user\project\verdaccio.master\website\i18n\fr-FR\docusaurus-plugin-content-docs\current\google-cloud.md" with unexpected output:
``

Reply:
We improved this behavior at https://github.com/facebook/docusaurus/pull/6937. Are you using the latest version?

Reply:
No yet, I haven't catching up yet :) but thanks for the heads up. Glad you improve it.